src/explore-education-statistics-frontend/src/app/layout.tsx

```diff
@@ -0,0 +1,91 @@
+// Import order is important - these should be at the top
+import '@frontend/polyfill';
+import '@frontend/loadEnv';
+import '../styles/_all.scss';
+import { NetworkActivityContextProvider } from '@common/contexts/NetworkActivityContext';
+import composeProviders from '@common/hocs/composeProviders';
+import { cookies } from 'next/headers';
+import Head from 'next/head';
+import React, { Suspense } from 'react';
+import { Metadata } from 'next';
+import { Dictionary } from '@common/types';
+import ApplicationInsightsContextProvider, {
+  ApplicationInsightsTracking,
+} from '../components/application-insights-context-provider';
+import QueryClientProvider from '../components/query-client-provider';
+import NavigationEvents from '../components/navigation-events';
+
+type Props = {
+  children: React.ReactNode;
+};
+
+const Providers = composeProviders(
+  ApplicationInsightsContextProvider,
+  NetworkActivityContextProvider,
+  QueryClientProvider,
+);
+
+export default function RootLayout({ children }: Props) {
+  const allCookies: Dictionary<string> = Object.assign(
+    {},
+    ...cookies()
+      .getAll()
+      .map(cookie => ({ [cookie.name]: cookie.value })),
+  );
+
+  return (
+    <html lang="en" className="govuk-template app-html-class">
+      <Head>
+        {process.env.NODE_ENV === 'development' && (
+          <>
+            {/* Force browser to not cache any assets */}
+            <meta httpEquiv="pragma" content="no-cache" />
+            <meta httpEquiv="cache-control" content="no-cache" />
+          </>
+        )}
+
+        <link
+          rel="icon"
+          sizes="48x48"
+          href="/assets/images/favicon.ico"
+          type="image/x-icon"
+        />
+        <link
+          rel="icon"
+          sizes="any"
+          href="/assets/images/favicon.svg"
+          type="image/svg+xml"
+        />
+        <link
+          rel="mask-icon"
+          href="/assets/images/govuk-icon-mask.svg"
+          color="#0b0c0c"
+        />
+        <link rel="apple-touch-icon" href="/assets/images/govuk-icon-180.png" />
+        <link rel="manifest" href="/assets/manifest.json" />
+
+        <meta
+          property="og:image"
+          content="/assets/images/govuk-opengraph-image.png"
+        />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
```

Comment:
Doesn't seem like any of these tags are actually making it into the `head`?

Seems like we don't even need this anymore as we can use Next's [new metadata functionality](https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead).

Reply:
I had noticed that, though I forgot to go back and address it so good spot! There's another SEO ticket about content in which I'm intending to migrate this across to the new metadata functionality, but thought I'd leave it for this PR. Will look into why they're not working though... (or just use the new metadata if quicker)